### DISCUSSION TITLE
I created a snippets.lua file in the lua/config folder, but it isn't loaded automatically

### DISCUSSION THREAD

**Discussion by apettel** [LazyVim/LazyVim#420] [General]
The documentation says: "The files under lua/config will be automatically loaded at the appropriate time, so you don't need to require those files manually."

Well, when i put a 
```lua
require("config.snippets")
```
in config/options.lua, then my snippets are loaded and everything works fine. But the documentation warns me about using "require" for files in the config folder: "DANGER - Do not require any files under lua/config/ or lazyvim.config manually. LazyVim will load those files automatically."

I am using nvim 0.9-dev and the current versions of all plugins.

**Comment by folke** [comment]
Only the files `options`, `autocmds` and `keymaps` are automatically loaded. Any other files you of course should require youreself

**Reply by apettel** [reply]
Thank you for the clarification! 

It would be nice, if the documentation would be updated:
```
"The files under lua/config will be automatically loaded at the appropriate time"
->
"The files autocmds.lua, keymaps.lua, lazy.lua and options.lua under lua/config will be automatically loaded at the appropriate time"
```

```
"DANGER - Do not require any files under lua/config/ or lazyvim.config manually."
->
"DANGER - Do not require autocmds, keymaps, lazy or options under lua/config/ or lazyvim.config manually."
```

**Reply by folke** [reply]
Feel free to create a PR! https://github.com/LazyVim/LazyVim

**Reply by apettel** [reply]
Ok, I created a PR for LazyVim/lazyvim.github.io
